content/800-data-platform/100-accelerate/580-local-development.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Local development'
+metaTitle: 'Accelerate: Local development'
+metaDescription: 'Learn how to use Prisma Accelerate in a development environment.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma Accelerate is designed to scale database connections efficiently in production environments. Prisma Accelerate also provides a global cache to reduce the load on your database and reduce query response time. To leverage Prisma Accelerate, it is essential to utilize a publicly accessible database.
+
+In development environments, you may want to use a local database to minimize expenses. Furthermore, you may consider extending Prisma Client with the Accelerate client extension once so that you can use a local database in development and a hosted database with Accelerate’s connection pooling and caching enabled. This eliminates the need for conditional logic to switch clients between development and production.
+
+This guide will explain how to use Prisma Accelerate client extension in a development environment with a local database.
+
+</TopBlock>
+
+## Using Prisma Accelerate client extension in development and production
+
+<br />
+<img src="images/accelerate-in-dev.png" alt="" />
+
+Accelerate doesn’t work with a local database. However, using a local database URL, you can add the Accelerate client extension to your Prisma Client. This will allow you to use a local database in your development environment. Keep in mind that, in this scenario, caching and connection pooling won't be available.
+
+Here is how to use a local PostgreSQL database with Prisma ORM and Prisma Accelerate client extension (without caching and connection pooling):
+
+1. Paste your local database URL in your .env file
+
+   ```.env
+   DATABASE_URL="postgres://username:password@127.0.0.1:5432/localdb"
+   ```
+
+2. Generate a Prisma Client
+
+   ```bash
+   npx prisma generate
+   ```
+
+   > Note: The `--no-engine` cannot be used when working with a local database. The command generates Prisma Client artifacts without a [Query Engine](/concepts/components/prisma-engines) file, which requires the use of an Accelerate connection string.
+
+3. Add Prisma Accelerate to your project
+
+   ```typescript
+   import { PrismaClient } from '@prisma/client'
+   import { withAccelerate } from '@prisma/extension-accelerate'
+
+   const prisma = new PrismaClient().$extends(withAccelerate())
+   ```
+
+   > The extended instance of Prisma Client will use the local database. Hence, Prisma Accelerate will not be used in your development environment to respond to your Prisma Client queries.
+
+<img src="images/accelerate-in-prod.png" alt="" />
+
+In a production environment, when a generated Accelerate connection string is used, you will be able to use connection pooling and caching by adding the Accelerate client extension to the `PrismaClient`.
```

Comment:
```suggestion
If an Accelerate connection string is used as the `DATABASE_URL` environment variable, Prisma Client will route your queries through Accelerate. We recommend using an Accelerate connection string in a production environment.
```

Feel free to adjust the text as you see fit

Reply:
@ruheni if they are using a Next.js edge runtime, they have no option but to use Accelerate as of now.